frontend/package.json

```diff
@@ -21,7 +21,7 @@
     "moment": "2.29.2",
     "moment-timezone": "0.5.33",
     "multiselect-react-dropdown": "2.0.21",
-    "npm": "8.1.0",
+    "npm": "8.11.0",
```

Comment:
This change is problematic because our frontend **does not build** with npm 8.11 due to some changes in peer dependency resolution. The latest we can use is 8.5.5